jobs/atc/templates/ldap_ca_cert.erb

```diff
@@ -0,0 +1 @@
+<%= p("ldap_auth.ca_cert", "") %>
```

Comment:
I think this'll need to be `ldap_auth.ca_cert.certificate` since the type for the property is `certificate`.

Reply:
@vito Hmm, I think it's fine like that. github_auth, cf_auth, etc. also only directly have `ca_cert`, I just copied their structure. Or is it wrong there?

Reply:
@JamesClonk They were all actually broken - https://github.com/concourse/concourse/pull/2479 fixed them. :)

Reply:
@vito changed it :+1: